test/metabase/upload_test.clj

```diff
@@ -813,32 +836,19 @@
 (deftest mysql-settings-test
   (testing "Ensure that local_infile is set to true for better MySQL testing"
     (mt/test-drivers [:mysql]
-      (with-mysql-local-infile-activated
-        (is (= "ON" (-> (sql-jdbc.conn/db->pooled-connection-spec (mt/db))
-                        (jdbc/query
-                         ["show global variables like 'local_infile'"])
-                        first
-                        :value)))))))
-
-(deftest load-from-csv-mysql-slow-way-test
```

Comment:
We don't need this test anymore, since we test with `local_infile` off throughout all the other tests now.